README.md

```diff
@@ -1 +1,191 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Travel Conference] <a name="about-project"></a>
+
+
+
+**[Travel Conference]** is a website that shows my skills in web development. It is a mock website for a travel conference that could happen in Kenya.<br>
+A [loom video](https://www.loom.com/share/d6172d4d8ba84125b7b3bc3a7e02d10c) description of my project.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.quackit.com/html/codes/html_code_library.cfm">Html</a></li>
+    <li><a href="https://www.w3docs.com/course/css-the-complete-guide-2020-incl-flexbox-grid-sass">css</a></li>
+  </ul>
+</details>
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Main Page]**-> The main landing page for the website.
+- **[About Page]**-> A webpage with a more detailed description of the conference.
+
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ Git and Code editor.
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/gaks1/capstone-project1.git
+
+### Install
+
+Install this project with:
+http server:
+
+npm install http-server -g
+http-server
+
+
+
+To run the project, execute the following command:
+
+http-server public
+
+
+
+### Run tests
+
+To run tests, go to:
+
+1. htmltester--> https://wave.webaim.org/
+2. csstester-->https://jigsaw.w3.org/css-validator/
+
+
+### Deployment
+
+You can deploy this project using:
+
+github pages.
+
+Deployed [project](https://gaks1.github.io/capstone-project1/) for view.
+ 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤  Isaac Gakure Wanyoike
+
+ - GitHub: [@githubhandle](https://github.com/gaks1)
+ - Twitter: [@twitterhandle](https://twitter.com/bopplov)
+-  LinkedIn: [LinkedIn](https://www.linkedin.com/in/isaac-wanyoike-1841a8172/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- **[Bug fix]**-> Fix bugs.
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/gaks1/Isaac-portfolio/issues).
```

Comment:
- Hey, please, let's kindly update the issues page to be that of this current project as this is not the current project's issues's page.